schema.sql

```diff
@@ -4,9 +4,8 @@ CREATE DATABASE therebasebook;
 \c therebasebook;
 
 -- SERIAL- adds not null constraint, should increment by 1 for each new entry
-DROP TABLE IF EXISTS users CASCADE;
 CREATE TABLE users (
-    id SERIAL PRIMARY KEY NOT NULL,
+    id SERIAL PRIMARY KEY NOT NULL UNIQUE,
```

Comment:
we don't need UNIQUE. It's redundant. Primary Keys are always unique.
https://www.w3resource.com/PostgreSQL/primary-key-constraint.php